jit.c

```diff
@@ -1231,6 +1239,7 @@ struct riscv_jit_t *rv_jit_init(uint32_t bits)
 void rv_jit_free(struct riscv_jit_t *jit)
 {
     if (jit->block_map) {
+        block_map_clear(jit->block_map);
         block_map_free(jit->block_map);
         jit->block_map->map = NULL;
```

Comment:
It looks like we had set  `jit->block_map->map` to `NULL` already in `block_map_free`, so the statement can be removed.

Reply:
Okay